pkgs/development/python-modules/restinstance/default.nix

```
@@ -0,0 +1,26 @@
+{ lib, fetchFromGitHub, buildPythonPackage, jsonschema, nox }:
```

Will make the diffs in the future easier to read.

```suggestion
{ lib
, fetchFromGitHub
, buildPythonPackage
, jsonschema
, nox
}:
```